web/impact/impact/models/utils.py

```diff
@@ -16,3 +18,17 @@ def model_has_field(model, field_name):
         return True
     except FieldDoesNotExist:
         return False
+
+
+def model_name_to_snake(value):
+    original_model_string = camel_case_to_spaces(value)
+    new_model_string = original_model_string.replace(" ", "_")
+    return new_model_string
+
+
+def snake_to_model_name(value):
+    old_value = value.split('_')
+    new_value = ""
+    for word in old_value:
+        new_value += (word[0].upper() + word[1:])
```

Comment:
In more idiomatic python, this would be:
```
def snake_to_model_name(value):
    pieces = value.split("_")
    return "".join(map(str.title, pieces))
```
or just 
```
def snake_to_model_name(value):
    return "".join(map(str.title, value.split("_")))
```

Reply:
Do we want to worry about the possibility of spaces existing in `value`? 
To me, that would be an invalid input and should throw an exception, but that might be over-thinking it.

Reply:
Updated.  Value will never have spaces as that isn't how the models are formatted.  They're always pseudo camel case or underscores.